update certification process (#201)
IBM/ibm-satellite-storage: README.md

# IBM Cloud Satellite storage
IBM Cloud Satellite Storage provides capabilities to manage the lifecycle of a storage solution on a ROKS cluster deployed in a Satellite Location. It simplifies the storage solution deployment and maintenance including security patches & version upgrades. The core objective is to simplify the storage life cycle management for Cluster Admins by hiding the complexity.

This repository is to collaborate with IBM Partners and Storage Solution Vendors to develop storage configuration templates for IBM Cloud Satellite.

## Overview

> With IBM Cloud Satellite, you can bring your own compute infrastructure that is in your on-premises data-centre, at other cloud providers, or in edge networks as a Satellite Location to IBM Cloud. Then, you use the capabilities of Satellite to run IBM Cloud services on this infrastructure, and consistently deploy, manage, and control your app workloads. For more information, see the IBM documentation.

Reviewer: ```suggestion
With IBM Cloud Satellite, you can bring your own compute infrastructure that is in your on-premises data center, at other cloud providers, or in edge networks as a Satellite Location to IBM Cloud. Then, you use the capabilities of Satellite to run IBM Cloud services on this infrastructure, and consistently deploy, manage, and control your app workloads. For more information, see the IBM documentation.
```

- [IBM Cloud Satellite](https://www.ibm.com/cloud/satellite)

> - [ROSK in IBM Cloud Satellite](https://cloud.ibm.com/docs/satellite?topic=satellite-managed-services)

Reviewer: ```suggestion
- [Red Hat OpenShift on IBM Cloud Satellite](https://cloud.ibm.com/docs/satellite?topic=satellite-managed-services)
```

- [Getting started](https://cloud.ibm.com/docs/satellite?topic=satellite-getting-started)
- [IBM Cloud Satellite Storage](https://cloud.ibm.com/docs/satellite?topic=satellite-sat-storage-template-ov)

## Satellite storage partner certification process
![Storage Template Registration Flow](./satellite-storage-registration-flow.png) 

> ## Register with IBM Cloud

Reviewer: ```suggestion
## Registering with IBM Cloud
```

1. [Create an IBMid](https://cloud.ibm.com/registration).

1. Create an issue under [IBM Satellite Storage](https://github.com/IBM/ibm-satellite-storage/issues) repo.
   1. Include the subject line: `<storage-solution-name> integration`*. For example: `LocalVolume integration`. 
   1. Provide a high level description of the storage solution with a link to the user documentation or public repository.

> ## Development environment

Reviewer: ```suggestion
## Setting up your development environment
```

> 1. Collaborate with **`IBM Cloud Satellite Storage Team`** to setup your Satellite Location for development and testing. 

Reviewer: ```suggestion
1. Collaborate with the **`IBM Cloud Satellite Storage Team`** to setup your Satellite Location for development and testing. 
```

> ## Test your storage solution with ROKS in IBM Cloud Satellite 

Reviewer: ```suggestion
## Testing your storage solution in IBM Cloud Satellite 
```

1. [Create an IBMid](https://cloud.ibm.com/registration).

1. Create an issue under [IBM Satellite Storage](https://github.com/IBM/ibm-satellite-storage/issues) repo.

>    1. Include the subject line: `<storage-solution-name> integration`*. For example: `LocalVolume integration`. 

Reviewer 2: remove extra `*`

Author: remove the character

   1. Provide a high level description of the storage solution with a link to the user documentation or public repository.

> ## Registering with IBM Cloud

Reviewer 2: Why same heading at 2 places? line 17 & 25?

Author: Fixed the heading

> 1. Collaborate with **`IBM Cloud Satellite Storage Team`** to setup your Satellite Location for devlopment and testing. 

Reviewer 2: `devlopment` ----> `development`

Author: fixed the typo

   ```
1. Test your deployment.
   1. From `IBM Cloud Console` -> `Satellite` -> `Configurations` -> `Create Configuration`
   1. From `IBM Cloud Console` -> `Satellite` -> `Configurations` -> `Select the Configuration` -> `Add version and upload the deployment yaml`
   1. From the `IBM Cloud Console` -> `Satellite` -> `Configurations` -> `Select the Configuration` -> `Create Subscription`
   1. Verify that the resources are deployment to the cluster. 
   1. Create a PVC and run the sniff test using the IBM Provides test bucket to make sure deployment is working as expected.
 
1. After you have verified your template in the development environment, update the issue. Include the following information in your issue and ensure that you remove any secrets from your files.
   1. Your `deployment.yaml` file.
   1. An example`pvc.yaml` file.
   1. An example `app.yaml`.
   1. The output of the `oc get sc` command that displays the storage classes your deployment creates.
   1. The output of the `oc get -n <namespace> pvc` command that displays the PVCs your deployment creates.
   1. The output of the `oc get -n <namespace> pods` command that displays the pods your deployment creates.

> ## Solution validation and certification

Reviewer: ```suggestion
## Validating and certifying your solution
```

> IBM Team will collaborate with you to ensure your product works well with the IBM Cloud Paks. The intent of this certification is for our two companies to produce enterprise grade, best of breed, resilient, scalable solutions driving toward a consistent and maintainable deployed environment.

Reviewer: ```suggestion
The IBM storage team will collaborate with you to ensure your product works well with the IBM Cloud Paks. The intent of this certification is for our two companies to produce enterprise grade, best of breed, resilient, scalable solutions driving toward a consistent and maintainable deployed environment.
```

2. Add the parameter to your `custom-parameters.json` file in the following format. You can also include a default value for the parameter.
   ```json
      {
         "description": "A description of the parameter.",
         "displayname": "The display name for the Satellite UI",
         "name": "devicepath",
         "default": "/dev/sdc",
         "regex": "^/dev/[a-zA-Z0-9]+$",
         "max-length": "64",
         "required": "true",
         "category": "config",
         "obfuscate": "false",
         "type": "text"
      }
   ```

> #### Custom parameter reference

Reviewer: ```suggestion
## Understanding the Satellite storage template parameters
```
